__tests__/kata4.humanCatDogYears.test.js

```diff
@@ -1,3 +1,9 @@
-const { humanCatDogYears } = require('../src');
+const humanCatDogYears = require('../src/kata4.humanCatDogYears.js');
 
-// Look Ma, no handlebars!!!
+describe('humanCatDogYears', () => {
+    it('returns an array of human years cat years and dog years', () => {
+      expect(humanCatDogYears(12)).toEqual([12, 64, 74])
+      expect(humanCatDogYears(4)).toEqual([4, 32, 34])
+      expect(humanCatDogYears(7)).toEqual([7, 44, 49])
```

Comment:
Good work adding a range of test cases. It's also important to test for "edge cases". What would happen if you passed this function a 1, 0 or even a negative number?